site/app/templates/forum/EditImgTable.twig

```diff
@@ -0,0 +1,26 @@
+{% if post_attachment.params is defined %}
+    <div id="display-existing-attachments">
+        <table class="file-upload-table edit-imgs-table">
+            <thead>
+                <tr>
+                    <th>Attached Images</th>
+                </tr>
+            </thead>
+            <tbody>
+            {% for row in post_attachment.params.unencoded_data %}
```

Comment:
Can you choose a more descriptive variable name here?  `row` is a pretty common variable name, and it's easy for developers to get confused if this code gets more complicated in the future.